tests/runtest.proj

```diff
@@ -288,10 +288,14 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
                     {
                       sErrorText = "Unable to read error file: " + errorFile%3B
                     }
+                    System.IO.StreamReader outputReader = new System.IO.StreamReader(outputFile)%3B
```

Comment:
I think we should have a try/catch block around reading this file. Please follow the pattern for the errorFile.